src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs

```diff
@@ -227,7 +227,7 @@ void InitializeShortcutBar()
         {
             ShortcutToolbar shortcutBar = new ShortcutToolbar(this.dynamoViewModel.Model.UpdateManager);
             shortcutBar.Name = "ShortcutToolbar";
-
+            Library.Text = Properties.Resources.LibraryText;
```

Comment:
OK. Now I understand why you need a name for this text box. No, it is not necessary to do that. Search `xmlns:p` in .xaml files and find out how the text could be bound to resource string in xaml file.